setup-mac.py

```diff
@@ -0,0 +1,60 @@
+"""
+Setup file for mimic
+"""
+
+from setuptools import setup, find_packages
+from twisted.plugin import getPlugins, IPlugin
+from py2app.build_app import py2app
+from mimic import plugins
+
+
+NAME = 'mimic'
+VERSION = '0.1'
+ID = 'mimic'
+SCRIPT='bundle/start-app.py'
+TEST_SCRIPT='bundle/run-tests.py'
+PLIST = dict(
+    CFBundleName                = NAME,
+    CFBundleShortVersionString  = ' '.join([NAME, VERSION]),
+    CFBundleGetInfoString       = NAME,
+    CFBundleExecutable          = NAME,
+    CFBundleIdentifier          = 'com.yourdn.%s' % ID,
```

Comment:
`CFBundleIdentifier` should be, roughly, a domain name that you own (or at least in a TLD that you own).  I guess we could register a domain for Mimic :) but I think com.rackspace.mimic would be an OK starting point.